persistence/sqlite/db.go

```diff
@@ -53,18 +55,16 @@ func (s *sqlite) Save(key int64, value []byte) (int64, error) {
 	return key, nil
 }
 
-func (s *sqlite) Get(key int64) ([]byte, error) {
-	stmt, err := s.Prepare(getKey)
+func (s *sqlite) Get(keys []int64) ([]persistence.JSONValue, error) {
+	stmt, err := s.Prepare(getKeys)
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	var value []byte
-	err = stmt.QueryRow(key).Scan(&value)
-	if err != nil {
-		log.Println(err)
-	} else {
-		log.Printf("Fetched for key %v", key)
+	var value []persistence.JSONValue
+	err = stmt.QueryRow(util.SliceToString(keys)).Scan(&value)
```

Comment:
Any chance we can avoid SliceToString()? That is, simply pass an array as the variable?

Reply:
I tried, it seems the driver cannot really handle arrays